.github/workflows/spotless_checks.yml

```diff
@@ -0,0 +1,17 @@
+name: Spotless Checks
+
+on:
+  schedule:
+    - cron: "0 5 * * *"
```

Comment:
When I said `5 AM`, I meant it in IST. I think this will run at 5 AM UTC. Could you update this so it runs in IST, please?

Reply:
At this moment there is no time zone parameter. So I can just change the time to 11:30 PM UTC

Reply:
Yup, that works.